packages/@vue/cli-ui/apollo-server/connectors/folders.js

```diff
@@ -13,7 +13,7 @@ const pkgCache = new LRU({
 
 const cwd = require('./cwd')
 
-function isDirectory (file) {
+function isDirectorySync (file) {
```

Comment:
Don't know if replacing this function to fully sync will break at other place, please advice if is save to use the async version